src/plugins/ui_feature/insert_pivot.ts

```diff
@@ -131,7 +131,7 @@ export class InsertPivotPlugin extends UIPlugin {
       tableType: "dynamic",
       sheetId,
       ranges: [this.getters.getRangeDataFromZone(sheetId, zone)],
-      config: { ...PIVOT_TABLE_CONFIG, numberOfHeaders },
+      config: { ...PIVOT_TABLE_CONFIG, numberOfHeaders, automaticAutofill: true },
```

Comment:
Maybe add the boolean directly inside `PIVOT_TABLE_CONFIG` ? It probably make more sense. And I'm pretty sure the test `Inserted pivot is inserted with a table` will fail in Odoo otherwise.